# The status of a tracking algorithm

### the status by itself
int8 status

### the predefined constants
int8 NO_TARGET = 1
int8 TARGET_TRACKING_OK = 2
int8 TARGET_TRACKING_LOST = 3
